src/main/java/com/thoughtworks/forecastAlexaSkillBackend/model/Cart.java

```diff
@@ -0,0 +1,24 @@
+package com.thoughtworks.forecastAlexaSkillBackend.model;
+
+import java.util.Date;
+import java.util.List;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+public class Cart {
+
+  private String id;
+
+  private Date orderDate;
+
+  private User user;
+
+  private Company company;
+
+  private List<ShoppingItem> cartItems;
```

Comment:
Instead of ShoppingItem, lets use CartItem model extending ShoppingItem and add new field called "totalProce"

Reply:
I think total price will be the feature of the cart not cartItem